.well-known/apple-app-site-association

```diff
@@ -1,30 +1 @@
-{
-  "applinks": {
-    "details": [
-      {
-        "appIDs": [ "NZ6PH75U7K.com.rainydayapps.Blockchain" ],
-        "components": [
-          {
-            "/": "/login/*",
-            "comment": "Matches any URL whose path starts with /login/"
-          },
-          {
-            "/": "/#/login/*",
-            "comment": "Matches any URL whose path starts with /#/login/"
-          },
-          {
-            "/", "/signup/*",
-            "comment": "Matches any URL whose path starts with /signup/"
-          },
-          {
-            "/", "/#/signup/*",
-            "comment": "Matches any URL whose path starts with /#/signup/"
-          }
-        ]
-      }
-    ]
-  },
-  "webcredentials": {
-    "apps": [ "NZ6PH75U7K.com.rainydayapps.Blockchain" ]
-  }
-}
\ No newline at end of file
+{"applinks":{"apps":[],"details":[{"appID":"NZ6PH75U7K.com.rainydayapps.Blockchain","paths":["NOT /_/*","/*"]}]}}
```

Comment:
This looks scary, was this intended?

Reply:
also file missing empty line at the end

Reply:
Yep, this is the ios mobile site association file, so smart links will open the ios app. iOS team asked me to update it. 

Line added.